fog/ledger/enclave/impl/src/lib.rs

```diff
@@ -252,41 +265,30 @@ where
             .into_iter()
             .map(|(responder_id, query_response)| {
                 let plaintext_bytes = self.ake.backend_decrypt(&responder_id, &query_response)?;
```

Comment:
High level question - which aspects need to be constant time?

Reply:
I believe this was discussed in the live walkthrough, at this time `oblivious_utils::collate_shard_key_image_search_results` is the constant time behavior. It's behavior should not have changed in this PR.